scripts/mkcloud

```diff
@@ -1253,7 +1253,14 @@ function sanity_checks
     fi
 
     case $cinder_backend in
-        ''|netapp|local|nfs|raw|rbd)
+        ''|netapp|local|raw|rbd)
+            ;;
+        nfs)
+            if [[ ! " ${steplist[*]} " == *" setuplonelynodes "* ]] ||
```

Comment:
The name **step** was chosen on purpose. It should still be possible to run mkcloud with any single step, one after the other, each started separately and do the needed actions on the cloud.
This check will make it impossible if the cinder_backend is nfs. Because if you manually step though it, then the step list will only match this test for this very step.

So, steplist != sequence  :)